image-layout.md

```diff
@@ -5,7 +5,7 @@ This layout MAY be used in a variety of different transport mechanisms: archive
 
 Given an image layout and a ref, a tool can create an [OCI Runtime Specification bundle](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/bundle.md) by:
 
-* Following the ref to find a [manifest](manifest.md#image-manifest), possibly via a [manifest list](manifest-list.md#manifest-list)
+* Following the ref to find a [manifest](manifest.md#image-manifest), possibly via a [image index](image-index.md#manifest-list)
```

Comment:
“a image index” → “an image index”?

And do we really need to call this an “image index”?  Can't it just be an “index”?  There doesn't seem to be much specific to images about an array of descriptors.